docker-compose.yml

```diff
@@ -84,6 +84,8 @@ services:
       - UWSGI_DIE_ON_TERM=1
       - UWSGI_MODULE=signals.wsgi:application
       - DJANGO_SETTINGS_MODULE=signals.settings.development
+      - SIGMAX_AUTH_TOKEN
```

Comment:
Is this really needed here? We've a lot of custom ENV settings that we don't expose here. Just give them a sane default in the `settings` module.